src/main/scala/net/snowflake/spark/snowflake/SnowflakeWriter.scala

```diff
@@ -87,6 +90,35 @@ private[snowflake] class SnowflakeWriter(jdbcWrapper: JDBCWrapper) {
     io.writeRDD(sqlContext, params, strRDD, output.schema, saveMode, format)
   }
 
+  def afterColumnMapping(schema: StructType,
```

Comment:
let's change this function name again. function name usually starts with verb.
for example, `mapColumns`